conda-recipes/wasabi-feedstock/recipe/meta.yaml

```diff
@@ -0,0 +1,47 @@
+# adapted from the conda-forge wasabi recipe
+{% set name = "wasabi" %}
+{% set version = "0.3.0" %}
+{% set sha256sum = "051fd33b47cf5f21626a3b750001b267c464c84a33f8e8fbcb49e48c8fe82838" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256sum }}
+build:
+  linxu-ppc64le: python
```

Comment:
Problem here.

For noarch packages, we'll want: `noarch: python` here.

Reply:
done